diesel_cli/src/infer_schema_internals/sqlite.rs

```diff
@@ -210,7 +210,14 @@ pub fn determine_column_type(
         String::from("Float")
     } else if is_double(&type_name) {
         String::from("Double")
-    } else if type_name == "datetime" || type_name == "timestamp" {
+    } else if type_name == "datetime" {
+        // Decision made with the help of https://www.sqlite.org/lang_datefunc.html
+        if type_name.contains("localtime") || type_name.contains("utc") {
```

Comment:
I would be also fine with just not generating the sqlite specific timestamptz variant at all in diesel_cli. This would allow users to manually change their schema if they are sure that this is the correct type. Otherwise this is just not used yet and we may figure out a better way layer on.